aptos-move/framework/aptos-framework/sources/Coin.move

```diff
@@ -85,8 +86,56 @@ module AptosFramework::Coin {
         borrow_global<CoinInfo<CoinType>>(coin_address).supply
     }
 
+    /// Returns the `value` of the passed in `coin`. 
+    public fun value<CoinType>(coin: &Coin<CoinType>): u64 {
+        coin.value
+    }
+
+    /// Returns `true` if the type `CoinType` is a registered coin.
+    /// Returns `false` otherwise.
+    public fun is_coin<CoinType>(): bool {
+        let type_info = TypeInfo::type_of<CoinType>();
+        let coin_address = TypeInfo::account_address(&type_info);
+        exists<CoinInfo<CoinType>>(coin_address)
+    }
+
     // Public functions
 
+    /// Create a new `Coin<CoinType>` with a value of `0`.
```

Comment:
nit we probably should move the // Public functions properly

Reply:
I ordered alphabetically functions in recent commit, is that what did you mean?